multinet/api/views/workspace.py

```diff
@@ -52,7 +53,7 @@ def get_queryset(self):
         public_workspaces = Q(public=True)
         return self.queryset.filter(
             public_workspaces | readable_private_workspaces | owned_workspaces
-        )
+        ).distinct()
```

Comment:
Ah! Yes it should. I put that in while testing after you described the bug to me and forgot to remove it. I can take it out and merge master into this branch.

Reply:
In general, as long as changes made on a branch that are also on the base don't conflict with each other, there's no need to remove the changes from the branch before merging in the base. I find (again, in general) that it's better not to worry about that sort of thing and allow the conflict resolution algorithms to help you out by not making you edit history, etc., just because you had the same good idea that someone else did earlier (or simultaneously).